webui/src/pages/repositories/repository/actions/index.jsx

```diff
@@ -138,7 +125,7 @@ const ActionsList = ({ repo, after, onPaginate, branch, commit, onFilterBranch,
     if (!!error) content = <Error error={error}/>
 
     else if (loading) content = <Loading/>
-    else if (results.length === 0 && !nextPage) content = <GetStarted />
+    else if (results.length === 0 && !nextPage) content = <Alert variant="info" className={"mt-3"}>No actions have been logged yet.</Alert>
```

Comment:
```suggestion
    else if (results.length === 0 && !nextPage) content = <Alert variant="info" className={"mt-3"}>No actions configured.</Alert>
```

(Most important to me here is not to say "logged", I find that verb confusing in this context.)